lib/matplotlib/tests/test_contour.py

```diff
@@ -292,6 +293,15 @@ def test_contourf_decreasing_levels():
     assert_equal(len(w), 2)
 
 
+@cleanup
+def test_contourf_symmetric_locator():
+    # github issue 7271
+    z = np.arange(12).reshape((3,4))
```

Comment:
in case you don't feel like parsing the Travis logs, the pep8 checkers wants a space after the comma here:

` z = np.arange(12).reshape((3, 4))`